Wrappers/Python/test/test_algorithms.py

```diff
@@ -1084,7 +1084,7 @@ def test_SPDHG_vs_PDHG_implicit(self):
         pdhg = PDHG(f=f, g=g, operator=operator, tau=tau, sigma=sigma,
                     max_iteration=1000,
                     update_objective_interval=500)
-        pdhg.run(1000,verbose=0)
+        pdhg.run(100,verbose=0)
 
         subsets = 10
```

Comment:
Reducing the number of subsets will also speed things up e.g. 5 subsets

Reply:
In fact reducing the `TotalVariation` iterations has more effect